lib/chewy/type/import.rb

```diff
@@ -84,19 +84,26 @@ def bulk_body(action_objects, indexed_objects = nil)
           end
         end
 
+        # Returns the value of the identifier for an object
```

Comment:
well, this module is included to Type, so type is actually self here and you can use `identify` directly from here